AEM/optimum-habana-perf.md

```diff
@@ -1,48 +1,57 @@
-It is recommended to use the PyTorch Docker image to run the examples below.
+It is recommended to use the Optimum-Habana fp8 Benchmark [Dockerfile](https://github.com/HabanaAI/Gaudi-tutorials/blob/main/PyTorch/Hugging_Face_pipelines/Benchmarking_on_Optimum-habana_with_fp8/Dockerfile) to run the examples below.
 
 To use the provided Dockerfile for the sample, follow the [Docker Installation guide](https://docs.habana.ai/en/latest/Installation_Guide/Additional_Installation/Docker_Installation.html) to setup the Habana runtime for Docker images.  
-The Docker image assists in setting up the PyTorch software and packages to run the samples. However, installing additional required packages like DeepSpeed is still necessary to run the samples. 
 
-### Get examples from optimum-habana github repository
-To benchmark Llama2 and Llama3 models, obtain optimum-habana from the GitHub repository using the following command.
-```bash
-git clone -b v1.16.0 https://github.com/huggingface/optimum-habana.git
-cd optimum-habana/examples/text-generation
-```
+## Build and Run the Benchmark Docker instance 
+
+### Get Dockerfile and Benchmark scripts
+First, get the Dockerfile and Benchmark scripts from Gaudi-Tutorial GitHub Repositroy following below command.  
 
-### Docker Run
-After building the Docker image, run the following command to start a Docker instance, which will open in the text-generation folder inside the docker instance.
 ```bash
-docker run -it --runtime=habana -e HABANA_VISIBLE_DEVICES=all -e OMPI_MCA_btl_vader_single_copy_mechanism=none   --cap-add=ALL --privileged=true  --net=host --ipc=host  -v "$PWD/../../":/workspace --workdir  /workspace/examples/text-generation  vault.habana.ai/gaudi-docker/1.20.0/ubuntu24.04/habanalabs/pytorch-installer-2.6.0:latest
+git clone https://github.com/HabanaAI/Gaudi-tutorials.git
+cd Gaudi-tutorials/PyTorch/Hugging_Face_pipelines/Benchmarking_on_Optimum-habana_with_fp8
 ```
-{:.note}**NOTE:** The Huggingface model file size might be large, so it is recommended to use an external disk as the Huggingface hub folder. Export the HF\_HOME environment variable to the external disk and then export the mount point into the Docker instance. ex: "-e HF\_HOME=/mnt/huggingface -v /mnt:/mnt"
-
-### Install required packages inside docker
-First, install the optimum-habana:
+### Docker Build
+To build the image from the Dockerfile, please follow below command to build the optimum-habana-text-gen image.
 ```bash
-pip install --upgrade-strategy eager optimum[habana]
+docker build --no-cache -t optimum-habana-text-gen:latest --build-arg https_proxy=$https_proxy --build-arg http_proxy=$http_proxy -f Dockerfile .
 ```
-
-Second, install the requirements:
+### Docker Run
+After docker build, users could follow below command to run and docker instance and users will be in the docker instance under text-generation folder.
```

Comment:
Change to: "After building the Docker image, users can use the command below to run a Docker instance, which will place them in the text-generation folder within the instance."